controllers/kubevirtmachine_controller.go

```diff
@@ -263,27 +269,54 @@ func (r *KubevirtMachineReconciler) reconcileNormal(ctx *context.MachineContext)
 	// Update the condition BootstrapExecSucceededCondition
 	conditions.MarkTrue(ctx.KubevirtMachine, infrav1.BootstrapExecSucceededCondition)
 
+	ipAddress := externalMachine.Address()
+	ctx.Logger.Info(fmt.Sprintf("KubevirtMachine %s: Got ipAddress <%s>", ctx.KubevirtMachine.Name, ipAddress))
+	if ipAddress == "" {
+		ctx.Logger.Info(fmt.Sprintf("KubevirtMachine %s: Got empty ipAddress, requeue", ctx.KubevirtMachine.Name))
+		return ctrl.Result{RequeueAfter: 20 * time.Second}, nil
+	}
```

Comment:
ah, Interesting. Maybe at some point we should consider watching VMIs for this reconcile controller.